lib/utils/utils.ts

```diff
@@ -208,3 +210,23 @@ export const base64ToHex = (b64: string) => {
 export const hexToUint8Array = (hex: string) => {
   return Uint8Array.from(Buffer.from(hex, 'hex'));
 };
+
+/** Converts input to EIP55 format.
+ * Prints the ith digit in uppercase if it's a letter and the 4*ith bit of the hash of the lowercase hexadecimal address is 1
+ * otherwise prints it in lowercase.
+ * Example: '0xfb6916095ca1df60bb79ce92ce3ea74c37c5d359' is converted to '0xfB6916095ca1df60bB79Ce92cE3Ea74c37c5d359'
+ */
+export const toChecksumAddress = (address: string) => {
```

Comment:
I'm thinking it might be more clear to call this `toEip55Address`?

Reply:
I agree.